### PR TITLE
Issue 449 - email to users who RSVP for events

### PR DESCRIPTION
**Pull request by BradHN1**

### CODE REVIEW

**Comment by BradHN1** [comment]
Here is an example email sent from the back end:

[Thank you for registering for Your Sustainable Future Home Now.pdf](https://github.com/massenergize/api/files/8226885/Thank.you.for.registering.for.Your.Sustainable.Future.Home.Now.pdf)

**Review by Opoku-Agyemang — APPROVED** [review]

File: src/api/store/event.py [hunk 1]
```diff
@@ -257,6 +253,8 @@ def create_event(self, context: Context, args) -> Tuple[dict, MassEnergizeAPIErr
         if not community:
           return None, CustomMassenergizeError("Please provide a valid community_id")
 
+      print("before event .create()")
```
Comment (Davidjwkinghorn) [inline src/api/store/event.py:256]: Not super important, but these print debugs should be removed when this PR gets merged.

File: src/database/models.py [hunk 2]
```diff
@@ -1609,18 +1612,18 @@ def info(self):
 
     def simple_json(self):
         data = model_to_dict(
-            self, exclude=["tags", "image", "community", "invited_communities"]
+            self, exclude=["tags", "image", "community", "invited_communities", "user"]
         )
-        data["start_date_and_time"] = self.start_date_and_time
-        data["end_date_and_time"] = self.end_date_and_time
+        #data["start_date_and_time"] = self.start_date_and_time
```
Comment (Davidjwkinghorn) [inline src/database/models.py:1617]: Start date/time and end date/time for events should be included in simple_json, I think.

**Review by Davidjwkinghorn — CHANGES_REQUESTED** [review]
Looks good to me apart from the two very minor changes I mentioned.

File: src/database/models.py (hunk 2 above)
Comment (BradHN1) [reply src/database/models.py:1617]: I agree, but they are already included by virtue of the model_to_dict call right above it.

Comment (Davidjwkinghorn) [reply src/database/models.py:1617]: Okay, I was wondering about that. Sounds good then.

File: src/api/store/event.py (hunk 1 above)
Comment (BradHN1) [reply src/api/store/event.py:256]: right you are.  I've removed them

File: src/database/models.py (hunk 2 above)
Comment (BradHN1) [reply src/database/models.py:1617]: Removed those commented lines

**Comment by Davidjwkinghorn** [comment]
There's still a failed test, but it's something with teams, so it's outside the scope of this PR.

**Review by Davidjwkinghorn — APPROVED** [review]
LGTM

**Pull request merged**